src/main/java/org/casbin/jcasbin/model/FunctionMap.java

```diff
@@ -40,15 +47,16 @@
      */
     public void addFunction(String name, AviatorFunction function) {
         fm.put(name, function);
+        enforcer.resetExpressionEvaluator();
```

Comment:
The end user calls enforcer's addFunction() to add a function, reset it in that function.